content/_data/logo/mario.yml

```diff
@@ -0,0 +1,7 @@
+---
+name: 'Mario'
```

Comment:
Plumber? I am afraid that Mario may cause some legal concerns. IANAL

Reply:
The company with 'N' that shall not be named is, uh, very protective of their intellectual property.

Reply:
Just rename it? Or delete?